src/Components/Results/ProgramPage/ProgramPage.tsx

```diff
@@ -80,6 +82,7 @@ const ProgramPage = ({ program }: ProgramPageProps) => {
         {displayIconAndHeader(program)}
         {displayEstimatedValueAndTime(program)}
       </div>
+      {program.warning.default_message && <WarningMessage message={program.warning.default_message} />}
```

Comment:
Here we probably want to pass in warning in Translation type, so it can be translated.